### PR TITLE
Feature/pim 1718 verified grain harvested commodity totals grid database

### PR DESCRIPTION
**Pull request by mhostett**

### CODE REVIEW

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/cirras.cuws.03_06_00_00.ddl.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+{
+  "databaseChangeLog": [
+{ "changeSet": {"author":"mhostettler","id":"87","changes": [
+  {"sqlFile": {"relativeToChangelogFile": "true", "path": "ddl/tables/cuws.create.verified_yield_contract.sql"}},
+  {"sqlFile": {"relativeToChangelogFile": "true", "path": "ddl/tables/cuws.alter.verified_yield_contract_commodity.sql"}},
```
Comment (dpitk2222) [inline cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/cirras.cuws.03_06_00_00.ddl.json:6]: Wrong filename.

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql [hunk 2]
```diff
@@ -0,0 +1,77 @@
+CREATE TABLE cuws.verified_yield_contract_commodity(
+    verified_yield_contract_commodity_guid    varchar(32)       NOT NULL,
+    verified_yield_contract_guid              varchar(32)       NOT NULL,
+    crop_commodity_id                         numeric(9, 0)     NOT NULL,
```
Comment (dpitk2222) [inline cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql:4]: Do we need is_pedigree_ind as well? The source table declared_yield_contract_commodity actually stores its data at the Commodity-Pedigree level, so I had assumed that Verified Yield would as well?

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    harvested_acres                           numeric(14, 4),
+    harvested_acres_override                  numeric(14, 4),
+    stored_yield_default_unit                 numeric(14, 4),
+    sold_yield_default_unit                   numeric(14, 4),
+    production_guarantee                      numeric(14, 4),
```
Comment (dpitk2222) [inline cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql:9]: Are we sure about storing production_guarantee? Does that mean we'll need to add it to the data sync? I had been thinking we might just load it dynamically whenever the Verified Yield screen is displayed.

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql (hunk 2 above)
Comment (mhostett) [reply cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql:4]: I'll ask Andrew

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql (hunk 3 above)
Comment (mhostett) [reply cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql:9]: I would like to use the data sync if we can and move away from accessing CIRRAS at runtime.
I think we should add it to the data sync considering that the PY and deductible level has to be brought over as well. PY is actually used to calculate some values in the yield summary table.

Comment (dpitk2222) [reply cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql:9]: Ok, sounds good.

File: cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql (hunk 2 above)
Comment (mhostett) [reply cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql:4]: We need to add pedigree. I'll update the documentation and release scripts

**Pull request merged**